vaadin-map-flow-parent/vaadin-map-flow-integration-tests/src/main/java/com/vaadin/flow/component/map/demo/TileWMSDemo.java

```diff
@@ -0,0 +1,44 @@
+package com.vaadin.flow.component.map.demo;
+
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.map.Map;
+import com.vaadin.flow.component.map.configuration.Coordinate;
+import com.vaadin.flow.component.map.configuration.layer.TileLayer;
+import com.vaadin.flow.component.map.configuration.source.TileWMSSource;
+import com.vaadin.flow.router.Route;
+
+import java.util.HashMap;
+
+/**
+ * Demo that displays a tile WMS layer on top of the default OSM background
+ * layer
+ * <p>
+ * Adapted from https://openlayers.org/en/latest/examples/wms-tiled.html
+ */
+@Route("vaadin-map/demo/tile-wms")
+public class TileWMSDemo extends Div {
```

Comment:
Since the ITs are really not that useful for testing if a source works correctly, I decided to start adding demos so that features can at least be tested manually in the future.